Donut_Shop.js

```diff
@@ -1,52 +1,81 @@
 
   (function () {
 
-    var topPotDonuts = function(franchise, options) {
+    var TopPotDonuts = function(franchise, options) {
       this.franchise = franchise;
       this.minCustomersPerHour = options.minCustomersPerHour;
       this.maxCustomersPerHour = options.maxCustomersPerHour;
       this.avrgDonutsPerCust = options.avrgDonutsPerCust;
-      this.myArray = [];
+      this.hourlyArray = []; //Stores individual hourlyTotal values for each iteration of loop
+      this.total;  //stores sum of hourlyrray values
     };
 
-   topPotDonuts.prototype.hourlyTotal = function() {
+
```

Comment:
Clean up the whitespace and formatting before you submit your work. In a professional setting this is way more important, as people need to be able to read code in a consistent way. Otherwise you might find your pull requests simply rejected.